modules/metastorage/src/integrationTest/java/org/apache/ignite/internal/metastorage/impl/ItIdempotentCommandCacheTest.java

```diff
@@ -475,46 +467,6 @@ public void testIdempotentCacheRestoreFromSnapshot(IdempotentCommand idempotentC
         }
     }
 
-    @Test
-    void testEvictIdempotentCommandsCacheAfterCompaction() {
```

Comment:
It's suspicious. Why it's removed?

Reply:
Is that correct that the only possibility to catch CompactionException on removeAll(commandIdsToEvict) is a removal attempt from an obsolete node, while another node already removed and compacted the commands?

Reply:
Commands that change metastorage never throws CompactedException.

Reply:
Yep, you are right!